src/components/map/LocationMarker.jsx

```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import { Marker, InfoWindow } from 'react-google-maps';
 import Button from '../button';
 
+const MARKER_ICON_MAPPINGS = {
+  blue: 'https://www.google.com/maps/vt/icon/name=assets/icons/spotlight/spotlight_pin_v2_shadow-1-small.png,assets/icons/spotlight/spotlight_pin_v2-1-small.png,assets/icons/spotlight/spotlight_pin_v2_dot-1-small.png,assets/icons/spotlight/spotlight_pin_v2_accent-1-small.png&highlight=000000,1400FF,FFFFFF,000000&color=1400FF?scale=1',
```

Comment:
Just to make sure, this is the official way Google exposes these markers? No CDN or anything?

Reply:
Correct, this is the official way to have custom colors on markers